src/pages/account/resetPassword.tsx

```diff
@@ -1,19 +1,61 @@
 import styled from '@emotion/styled';
-import type { NextPage } from 'next/types';
+import type { GetServerSideProps, NextPage } from 'next/types';
+import * as api from 'api';
 import { ResetPasswordForm } from 'components/account';
 import { Seo } from 'components/common';
+import { PAGE_PATH } from 'constants/common';
+import { getQueryParam } from 'utils';
 
-const ResetPassword: NextPage = () => {
+interface PageProps {
+  email: string;
+  token: string;
+}
+
+const ResetPassword: NextPage<PageProps> = ({
+  email,
+  token,
+}: {
+  email: string;
+  token: string;
+}) => {
   return (
     <>
       <Seo title={'비밀번호 재성정 | a daily diary'} />
       <ContentWrapper>
-        <ResetPasswordForm />
+        <ResetPasswordForm email={email} token={token} />
       </ContentWrapper>
     </>
   );
 };
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { query } = context;
+  const email = getQueryParam(query.email);
+  const token = getQueryParam(query.token);
```

Comment:
p3; 
항상 배열을 반환하니까 요렇게 해도 좋을 것 같아요!

```suggestion
  const [email] = getQueryParam(query.email);
  const [token] = getQueryParam(query.token);
```